CHANGELOG.md

```diff
@@ -1,81 +1,81 @@
 # Changelog
 
-## Unreleased
+## 3.9.0 (feature release)
 
 ### New features
 
-#### Add prefix and suffix for text input
+#### Add a prefix or suffix to a text input component
 
-You can now add a prefix or suffix element to the text input to help users enter things like currencies and measurements.
+You can now [use prefixes and suffixes in the text input component](https://design-system.service.gov.uk/components/text-input/#prefixes-and-suffixes) to help users enter things like currencies and measurements.
+
+This was added in [pull request #1269: Add input prefix and suffix](https://github.com/alphagov/govuk-design-system/pull/1269/files). Thanks to [@simonwhatley](https://github.com/simonwhatley) and the GOV.UK Coronavirus Services Team.
+
+#### Customise navigation in the header component
+
+If you use the [header component with navigation](https://design-system.service.gov.uk/components/header/#header-with-service-name-and-navigation), you can now:
+
+- customise the section's `aria-label` text
+- add navigation items without links
+
+##### Customise aria-label text
+
+You can use the new:
+
+- `navigationLabel` option to set the `aria-label` text for the navigation section
+- `menuButtonLabel` option to set the `aria-label` text for the button that hides or shows the navigation section on mobile
 
 For example:
 
 ```javascript
-{{ govukInput({
-  label: {
-    text: "Amount, in pounds"
-  },
-  prefix: {
-    text: "£"
-  }
+{{ govukHeader({
+    navigationLabel: "Custom navigation section aria-label",
+    menuButtonLabel: "Custom menu button aria-label"
 }) }}
 ```
 
-You shouldn't rely on prefixes or suffixes alone because people who use a screen reader won't see them. You should add any specific type of information you need to the input label or hint text as well.
-
-This was added in [pull request #1816: Add input prefix and suffix](https://github.com/alphagov/govuk-frontend/pull/1816).
+The default labels are now:
 
-#### Add HTML test fixtures
-You can use our test fixtures to check you're outputting the same HTML that GOV.UK Frontend uses.
+- **Navigation menu** for `navigationLabel`
+- **Show or hide navigation menu** for `menuButtonLabel`
 
-This was added in [pull request #1925: Generate fixtures.json files for components on build:package](https://github.com/alphagov/govuk-frontend/pull/1925)
-
-#### Add new brand colour for The Foreign, Commonwealth and Development Office (FCDO)
+This was added in pull requests:
 
-This was added in [pull request #1918: Add new brand colour for FCDO](https://github.com/alphagov/govuk-frontend/pull/1918).
+- [#1905: Set navigation and mobile menu labels of the header component with new options](https://github.com/alphagov/govuk-frontend/pull/1905)
+- [#1943: Change header menu button label](https://github.com/alphagov/govuk-frontend/pull/1943)
```

Comment:
This could be 
> - [#1943: Change header menu button label](https://github.com/alphagov/govuk-frontend/pull/1943) - thanks to [@domoscargin](https://github.com/domoscargin) for raising this issue